torchgeo/trainers/etci2021.py

```diff
@@ -0,0 +1,67 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""Custom trainer for the ETCI2021 dataset."""
+
+from typing import Any, Dict, cast
+
+import kornia.augmentation as K
+import torch
+from torch import Tensor
+
+from .segmentation import SemanticSegmentationTask
+
+
+# TODO: move this functionality into SemanticSegmentationTask and remove this class
+class ETCI2021SemanticSegmentationTask(SemanticSegmentationTask):
```

Comment:
We can move these augmentations to the DataModule and get rid of this class now

Reply:
out of scope of this PR for sure